command/debug/debug.go

```diff
@@ -433,12 +425,20 @@ func (c *cmd) captureLongRunning() error {
 			return c.captureTrace(s, timestampDir)
 		})
 	}
-	// Capture logs
 	if c.configuredTarget("logs") {
 		g.Go(func() error {
 			return c.captureLogs(timestampDir)
```

Comment:
We should definitely do something similar to logs it will be much cleaner, but probably in a separate PR